vitrina/identifiers/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class IdentifiersConfig(AppConfig):
+    default_auto_field = "django.db.models.BigAutoField"
+    name = "vitrina.identifiers"
```

Comment:
Let's keep consistent with other apps.py files and add a label as well:

```suggestion
    default_auto_field = "django.db.models.BigAutoField"
    name = "vitrina.identifiers"
    label = "vitrina_identifiers"
```